osquery/events/linux/auditdnetlink.cpp

```diff
@@ -668,6 +675,155 @@ bool AuditdNetlink::configureAuditService() noexcept {
   return true;
 }
 
+bool AuditdNetlink::clearAuditConfiguration() noexcept {
+  int seq = audit_request_rules_list_data(audit_netlink_handle_);
+  if (seq <= 0) {
+    VLOG(1) << "Failed to list the audit rules";
+    return false;
+  }
+
+  // Attempt to list all rules
+  std::vector<AuditRuleDataObject> rule_object_list;
+  auto timeout = std::time(nullptr) + 5;
```

Comment:
Do you mind changing out the `std::time` calls for the `osquery::getUnixTime` API?